### PR TITLE
Go Back Button, Position Details

### PR DESCRIPTION
**Pull request by mjoyce91**
- Abstracts the "Go Back" button to its own connected component
- Reorders data on the Position details page
- Fixes margin on Position details page per design comps

### CODE REVIEW

File: src/Components/ResultsMultiSearchHeader/ResultsMultiSearchHeader.jsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ class ResultsMultiSearchHeader extends Component {
     const { placeholder, filters, userProfile, filtersIsLoading } = this.props;
     const { q, defaultGrade, defaultBureau, [SKILL_PARAM]: skills } = this.state;
 
-    // format skill codes
+    // format Skill codes
```
Comment (burgwyn) [inline src/Components/ResultsMultiSearchHeader/ResultsMultiSearchHeader.jsx]: You totally regex'd the whole repo

File: src/Components/PositionDetailsItem/PositionDetailsItem.jsx [hunk 2]
```diff
@@ -16,17 +16,19 @@ editPocContent, editWebsiteContent }) => {
   const formattedTourEndDate = tourEndDate ? formatDate(tourEndDate) : NO_END_DATE;
 
   const formattedBureau = details.bureau || NO_BUREAU;
+  const formattedTOD = propOrDefault(details, 'post.tour_of_duty') || NO_TOUR_OF_DUTY;
 
-  const formattedIncumbent = propOrDefault(details, 'current_assignment.user', NO_USER_LISTED);
-
-  const formattedTOD = propOrDefault(details, 'post.tour_of_duty', NO_TOUR_OF_DUTY);
+  const postDifferential = getDifferentialPercentage(propOrDefault(details, 'post.cost_of_living_adjustment'), NO_POST_DIFFERENTIAL);
+  const dangerPay = getDifferentialPercentage(propOrDefault(details, 'post.cost_of_living_adjustment'), NO_POST_DIFFERENTIAL);
```
Comment (burgwyn) [inline src/Components/PositionDetailsItem/PositionDetailsItem.jsx]: Let's make a `NO_DANGER_PAY` or just use `DEFAULT_TEXT`

Comment (mjoyce91) [reply src/Components/PositionDetailsItem/PositionDetailsItem.jsx]: Missed this, thanks

Comment (mjoyce91) [reply src/Components/PositionDetailsItem/PositionDetailsItem.jsx]: Updated

**Pull request merged**